scalene/find_browser.py

```diff
@@ -0,0 +1,12 @@
+def find_browser():
+    """Find the default browser if possible and if compatible."""
+    import webbrowser
+
+    # Mostly taken from the standard library webbrowser module. Search "console browsers" in there.
+    # In general, a browser belongs on this list of the scalene web GUI doesn't work in it.
+    # See https://github.com/plasma-umass/scalene/issues/723.
+    incompatible_browsers = {"www-browser", "links", "elinks", "lynx", "w3m", "links2", "links-g"}
+
+    browser = webbrowser.get()
```

Comment:
One thing to notice here is that we are no longer checking if `webbrowser.get()` is truthy. Looking at the code, it doesn't look like it will ever return None. It looks like it will raise `webbrowser.Error` if one cannot be found. If we want to handle that case I think we will need a try/catch.

Another reason besides reading the code itself to think that it cannot return `None` is that is how typeshed has typed it here: https://github.com/python/typeshed/blob/main/stdlib/webbrowser.pyi#L13C38-L13C38

Reply:
Agreed on the truthiness, thanks. How about degrading gracefully (in the face of an exception) to use the CLI?

Reply:
@emeryberger, good idea. I added a try/except in there.